content/en/tracing/setup_overview/setup/php.md

```diff
@@ -94,32 +94,43 @@ For other environments, please refer to the [Integrations][5] documentation for
 
 ### Install the extension
 
-Install the PHP extension using one of the [precompiled packages for supported distributions][5].
+Download the latest installer
 
-Once downloaded, install the package with one of the commands below.
+```shell
+curl -LO https://github.com/DataDog/dd-trace-php/releases/latest/download/datadog-setup.php
+```
+
+Run the installer
 
 ```shell
-# using RPM package (RHEL/Centos 6+, Fedora 20+)
-rpm -ivh datadog-php-tracer.rpm
+# Full installation: APM + ASM + Profiling (Beta)
+php datadog-setup.php --php-bin=all --enable-appsec --enable-profiling
 
-# using DEB package (Debian Jessie+ , Ubuntu 14.04+ on supported PHP versions)
-dpkg -i datadog-php-tracer.deb
+# APM only
+php datadog-setup.php --php-bin=all
 
-# using APK package (Alpine)
-apk add datadog-php-tracer.apk --allow-untrusted
+# APM + ASM
+php datadog-setup.php --php-bin=all --enable-appsec
+
+# APM + Profiling (Beta)
+php datadog-setup.php --php-bin=all --enable-profiling
 ```
 
-The extension will be installed for the default PHP version. To install the extension for a specific PHP version, use the `DD_TRACE_PHP_BIN` environment variable to set the location of the target PHP binary before installing.
+This command will install the extension to all the PHP binaries found in the host or container. If `--php-bin` is omitted then the installer runs in interactive mode asking the user to select the binaries for installation. The value of `--php-bin` can be a path to a specific binary in case `dd-trace-php` should be installed only to such binary.
 
-```shell
-export DD_TRACE_PHP_BIN=$(which php-fpm7)
-```
+Restart PHP (PHP-FPM or the Apache SAPI) and then visit a tracing-enabled endpoint of your application. View the [APM UI][5] to see the traces.
 
-Restart PHP (PHP-FPM or the Apache SAPI) and then visit a tracing-enabled endpoint of your application. View the [APM UI][6] to see the traces.
+<div class="alert alert-info">
+<strong>Note:</strong>
+It might take a few minutes before traces appear in the UI. If traces still do not appear after a few minutes, create a <a href="/tracing/troubleshooting/tracer_startup_logs?tab=php#php-info"><code>phpinfo()</code></a> page from the host machine and scroll down to the "ddtrace" section. Failed diagnostic checks will appear here to help identify any issues.
```

Comment:
```suggestion
It may take a few minutes before traces appear in the UI. If traces still do not appear after a few minutes, create a <a href="/tracing/troubleshooting/tracer_startup_logs?tab=php#php-info"><code>phpinfo()</code></a> page from the host machine and scroll down to the `ddtrace`. Failed diagnostic checks appear in this section to help identify any issues.
```